crates/type/src/digest/reader.rs

```diff
@@ -13,6 +13,7 @@ use sha2::digest::DynDigest;
 /// A hashing reader
 ///
 /// This type wraps another reader and hashes the bytes as they are read.
+#[allow(missing_debug_implementations)] // DynDigest does not implement Debug
```

Comment:
Same question as above re: manual Debug impl.

Reply:
Why? What would we print?